{"rendered": {"description": {"raw": "I occasionally get `[Errno 104] Connection reset by peer` when trying to upload a tarball with `s3cmd put` during a release.py call. It was just happening to me now with the ign-common2 prerelease. I noticed that [someone recommends](https://www.pimp-my-rig.com/2016/12/s3cmd-errno-104-connection-reset-by-peer.html) using `s3cmd sync` instead of `s3cmd put`, and that fixed their problem. I tried that, and it worked for me.", "markup": "markdown", "html": "<p>I occasionally get <code>[Errno 104] Connection reset by peer</code> when trying to upload a tarball with <code>s3cmd put</code> during a release.py call. It was just happening to me now with the ign-common2 prerelease. I noticed that <a data-is-external-link=\"true\" href=\"https://www.pimp-my-rig.com/2016/12/s3cmd-errno-104-connection-reset-by-peer.html\" rel=\"nofollow\">someone recommends</a> using <code>s3cmd sync</code> instead of <code>s3cmd put</code>, and that fixed their problem. I tried that, and it worked for me.</p>", "type": "rendered"}, "title": {"raw": "use s3cmd sync instead of s3cmd put to avoid [Errno 104] Connection reset by peer", "markup": "markdown", "html": "<p>use s3cmd sync instead of s3cmd put to avoid [Errno 104] Connection reset by peer</p>", "type": "rendered"}}, "type": "pullrequest", "description": "I occasionally get `[Errno 104] Connection reset by peer` when trying to upload a tarball with `s3cmd put` during a release.py call. It was just happening to me now with the ign-common2 prerelease. I noticed that [someone recommends](https://www.pimp-my-rig.com/2016/12/s3cmd-errno-104-connection-reset-by-peer.html) using `s3cmd sync` instead of `s3cmd put`, and that fixed their problem. I tried that, and it worked for me.", "links": {"decline": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/pullrequests/577/decline"}, "diffstat": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/diffstat/osrf/release-tools:6ec0a83db06b%0Daf4d75f4e417?from_pullrequest_id=577"}, "commits": {"href": "data/repositories/osrf/release-tools/pullrequests/577/commits.json"}, "self": {"href": "data/repositories/osrf/release-tools/pullrequests/577.json"}, "comments": {"href": "data/repositories/osrf/release-tools/pullrequests/577/comments_page=1.json"}, "merge": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/pullrequests/577/merge"}, "html": {"href": "#!/osrf/release-tools/pull-requests/577"}, "activity": {"href": "data/repositories/osrf/release-tools/pullrequests/577/activity.json"}, "diff": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/diff/osrf/release-tools:6ec0a83db06b%0Daf4d75f4e417?from_pullrequest_id=577"}, "approve": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/release-tools/pullrequests/577/approve"}, "statuses": {"href": "data/repositories/osrf/release-tools/pullrequests/577/statuses_page=1.json"}}, "title": "use s3cmd sync instead of s3cmd put to avoid [Errno 104] Connection reset by peer", "close_source_branch": true, "reviewers": [], "id": 577, "destination": {"commit": {"hash": "af4d75f4e417", "type": "commit", "links": {"self": {"href": "data/repositories/osrf/release-tools/commit/af4d75f4e417.json"}, "html": {"href": "#!/osrf/release-tools/commits/af4d75f4e417"}}}, "repository": {"links": {"self": {"href": "data/repositories/osrf/release-tools.json"}, "html": {"href": "#!/osrf/release-tools"}, "avatar": {"href": "data/bytebucket.org/ravatar/{41811d3d-19d0-428e-a134-976595e72508}ts=python"}}, "type": "repository", "name": "release-tools", "full_name": "osrf/release-tools", "uuid": "{41811d3d-19d0-428e-a134-976595e72508}"}, "branch": {"name": "default"}}, "created_on": "2018-05-04T21:18:52.760061+00:00", "summary": {"raw": "I occasionally get `[Errno 104] Connection reset by peer` when trying to upload a tarball with `s3cmd put` during a release.py call. It was just happening to me now with the ign-common2 prerelease. I noticed that [someone recommends](https://www.pimp-my-rig.com/2016/12/s3cmd-errno-104-connection-reset-by-peer.html) using `s3cmd sync` instead of `s3cmd put`, and that fixed their problem. I tried that, and it worked for me.", "markup": "markdown", "html": "<p>I occasionally get <code>[Errno 104] Connection reset by peer</code> when trying to upload a tarball with <code>s3cmd put</code> during a release.py call. It was just happening to me now with the ign-common2 prerelease. I noticed that <a data-is-external-link=\"true\" href=\"https://www.pimp-my-rig.com/2016/12/s3cmd-errno-104-connection-reset-by-peer.html\" rel=\"nofollow\">someone recommends</a> using <code>s3cmd sync</code> instead of <code>s3cmd put</code>, and that fixed their problem. I tried that, and it worked for me.</p>", "type": "rendered"}, "source": {"commit": {"hash": "dc97a1cc2284", "type": "commit", "links": {"self": {"href": "data/repositories/osrf/release-tools/commit/dc97a1cc2284.json"}, "html": {"href": "#!/osrf/release-tools/commits/dc97a1cc2284"}}}, "repository": {"links": {"self": {"href": "data/repositories/osrf/release-tools.json"}, "html": {"href": "#!/osrf/release-tools"}, "avatar": {"href": "data/bytebucket.org/ravatar/{41811d3d-19d0-428e-a134-976595e72508}ts=python"}}, "type": "repository", "name": "release-tools", "full_name": "osrf/release-tools", "uuid": "{41811d3d-19d0-428e-a134-976595e72508}"}, "branch": {"name": "s3cmd_sync"}}, "comment_count": 1, "state": "MERGED", "task_count": 0, "participants": [{"role": "PARTICIPANT", "participated_on": "2018-05-04T21:39:54.048863+00:00", "type": "participant", "approved": true, "user": {"display_name": "Jose Luis Rivero", "uuid": "{d12309b2-f745-42ee-b119-aec4fcdf81fe}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D"}, "html": {"href": "https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/109284c8b83411dbc7492138f6167e9ed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJR-5.png"}}, "nickname": "Jose Luis Rivero", "type": "user", "account_id": "557058:155a32e2-420c-4d50-98e0-0e722f63f906"}}], "reason": "", "updated_on": "2018-05-04T21:40:02.980659+00:00", "author": {"display_name": "Steve Peters", "uuid": "{2ccfed09-18b8-4921-8d58-15ef01092802}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D"}, "html": {"href": "https://bitbucket.org/%7B2ccfed09-18b8-4921-8d58-15ef01092802%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/1fb4816dad9e68337d3096f750951f6cd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSP-1.png"}}, "nickname": "Steven Peters", "type": "user", "account_id": "557058:5de38267-b118-494c-aa76-4fab35448816"}, "merge_commit": {"hash": "6ec0a83db06b", "type": "commit", "links": {"self": {"href": "data/repositories/osrf/release-tools/commit/6ec0a83db06b.json"}, "html": {"href": "#!/osrf/release-tools/commits/6ec0a83db06b"}}}, "closed_by": {"display_name": "Jose Luis Rivero", "uuid": "{d12309b2-f745-42ee-b119-aec4fcdf81fe}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D"}, "html": {"href": "https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/109284c8b83411dbc7492138f6167e9ed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJR-5.png"}}, "nickname": "Jose Luis Rivero", "type": "user", "account_id": "557058:155a32e2-420c-4d50-98e0-0e722f63f906"}}